runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CrypticClue.java

```diff
@@ -1899,4 +1911,38 @@ private static WorldPoint getViggoraLocation(ClueScrollPlugin plugin)
 				return null;
 		}
 	}
+
+	@SuppressWarnings("PMD.UnusedPrivateMethod")
+	private static String getGoldOreResourceAreaSolution(ClueScrollPlugin plugin)
+	{
+		return "Dig between the three gold ores in the Wilderness Resource Area. ".concat(getResourceAreaCost(plugin));
+	}
+
+	@SuppressWarnings("PMD.UnusedPrivateMethod")
+	private static String getSpeakToPilesSolution(ClueScrollPlugin plugin)
+	{
+		return "Speak to Piles in the Wilderness Resource Area.".concat(getResourceAreaCost(plugin));
+	}
+
+	private static String getResourceAreaCost(ClueScrollPlugin plugin)
+	{
+
+		if (plugin.getClient().getVarbitValue(Varbits.DIARY_WILDERNESS_ELITE) == 1)
+		{
+			return "No entry fee required.";
```

Comment:
is this wording needed? I think it's fine to add nothing about the coins if you have the elite diary done

Reply:
I'm fine with removing it, I left it because I think it's possible someone might not know/forget the elite diary removes this requirement and think they should grab some and "not know how much they need".

Reply:
Generally I don't think it's good to add more information than is needed; users can add clue notes if they wish, but I don't think it is necessary to mention a diary benefit that players would probably already know about.